server/context.go

```diff
@@ -141,6 +141,11 @@ func NewContext() *Context {
 	return ctx
 }
 
+// Insecure indicates whether or not SSL is to be used.
+func (ctx *Context) Insecure() bool {
```

Comment:
some of them (eg: `--attrs`) only apply to a single command. Still, that's an easy list to determine.